localstack/services/sqs/provider.py

```diff
@@ -254,27 +264,16 @@ def update_last_modified(self, timestamp: int = None):
 
         self.attributes[QueueAttributeName.LastModifiedTimestamp] = str(timestamp)
 
-    @property
-    def name(self):
-        return self.key.name
-
-    @property
-    def owner(self):
-        return self.key.account_id
-
-    @property
-    def arn(self) -> str:
-        return f"arn:aws:sqs:{self.key.region}:{self.key.account_id}:{self.key.name}"
```

Comment:
i think this should stay, i don't see the value in removing it. seeing the arn of the queue in the log messages is very useful.

Reply:
Added this back now that we have this info in the queue itself